routes/root.js

```diff
@@ -193,15 +193,52 @@ route.get('/claims/add', auth.ensureLoggedInGithub, (req, res) => {
   })
 })
 
-route.get('/claims/:id', auth.adminOnly, (req, res) => {
-  du.getClaimById(req.params.id)
-    .then(claim => {
-      if (!claim) throw new Error('No claim found')
-      res.render('pages/claims/id', { claim })
-    })
-    .catch(err => {
-      res.send('Error fetching claim id = ' + escapeHtml(req.params.id))
+route.get('/claims/:id', auth.adminOnly, async (req, res) => {
+  try {
+    const claimInfo = await du.getClaimById(req.params.id)
+    if (!claimInfo) {
+      throw new Error('Error finding claim')
+    }
+
+    const prSlug = new URL(claimInfo.pullUrl).pathname.replace(/\/+$/, '')
```

Comment:
Write a deserializer in utils to convert a url to:

{
project: "",
type: "pr" | "issue",
id: "",
}

Reply:
working on it